openmeter/entitlement/adapter/entitlement.go

```diff
@@ -412,14 +413,18 @@ func mapEntitlementEntity(e *db.Entitlement) *entitlement.Entitlement {
 				UpdatedAt: e.UpdatedAt.UTC(),
 				DeletedAt: convert.SafeToUTC(e.DeletedAt),
 			},
+			AnnotatedModel: models.AnnotatedModel{
```

Comment:
does it still need to be Annotated?

Reply:
I think the direction we agreed in general is that everything's supposed to have metadata, so yes?